db/R/query.base.R

```diff
@@ -70,7 +70,7 @@ query.base.con <- function(settings,...){
 ##' @param con database connection object
 ##' @return nothing, as a side effect closes all open connections
 ##' @author Rob Kooper
-##' @export
+##' @export query.close
```

Comment:
Why did you do this? You do not need to specify which function you are exporting -- roxygen knows it's the one the documentation is associated with. This introduces another place where the function name needs to be maintained.